server/command/connect.go

```diff
@@ -7,11 +7,34 @@ import (
 	"fmt"
 
 	"github.com/mattermost/mattermost-plugin-mscalendar/server/config"
+	"github.com/pkg/errors"
 )
 
 func (c *Command) connect(parameters ...string) (string, error) {
+	u, err := c.API.GetRemoteUser(c.Args.UserId) // needs fix
```

Comment:
Maybe not in this PR, but we should clean out `plugin.API` references and use `mscalendar.PluginAPI` instead.

Reply:
I mistakenly left this here. It should be using `withUserExpanded` `ExpandUser`, or `LoadUser`.